### DISCUSSION TITLE
Install package in pytorch environments

### DISCUSSION THREAD

**Discussion by dongyc98** [PaddlePaddle/PaddleSpatial#202] [BaiduKDDCup2022] [answered]
Could you install the following packages in the pytorch environment?
Darts, Kats, GluonTS, pytorch-forecasting, tsai, Flow Forecast, AtsPy, AutoTS

**Comment by ramber1836** [comment, marked as answer]
Most of the requested packages had been installed except for the packages of kats and flow forecast as they are incompatible with the current PyTorch environment.
